tele_giphy/game/tests/test_giphy.py

```diff
@@ -55,15 +55,17 @@ def test_functions(self, json_filename):
             assert item in resp.json()
 
     @responses.activate
-    def test_funnel(self, json_filename):
+    def test_giphy_call(self, json_filename):
```

Comment:
Better name! However it's still a bit unclear on what exactly it is testing and how it's different from the other test with the same name. Same goes for the other ones.